pages/docs/reference/mpp-dsl-reference.md

```diff
@@ -270,6 +268,99 @@ kotlin {
 
 </div>
 
+For binaries configuration, the following parameters are available:
+
+|**Name**|**Description**| 
+| --- | --- |
+|`compilation`|The compilation from which the binary is built. By default, `test` binaries are based on the `test` compilation while other binaries - on the `main` compilation.|
+|`linkerOpts`|Options passed to a system linker during binary building.|
```

Comment:
Is there any list of these options we could link to? Do developers know them?

Reply:
Doesn't seem so, system linkers are out of our scope. We just know that they have configuration options. All info is in their help/manuals.